components/dashboard/src/projects/Project.tsx

```diff
@@ -129,7 +129,7 @@ export default function () {
     }
 
     return <>
-        <Header title="Branches" subtitle={<h2 className="tracking-wide">View recent active branches for <a className="text-gray-500 hover:text-gray-800 font-semibold" href={project?.cloneUrl}>{project?.name}</a>.</h2>} />
+        <Header title="Branches" subtitle={<h2 className="tracking-wide">View recent active branches for <a className="gp-link" href={project?.cloneUrl!}>{toRemoteURL(project?.cloneUrl || '')}</a>.</h2>} />
```

Comment:
**issue(non-blocking)**: Replacing the project name with the repository URL could make long paths not scale well (see [example](https://gitlab.com/gitlab-com/marketing/community-relations/contributor-program/general/)). However, this a) is probably an edge case for now and b) could be resolved once we have an overview page where we list full details for a project.

<img width="1349" alt="Screenshot 2021-09-29 at 11 19 59 AM" src="https://user-images.githubusercontent.com/120486/135231473-0f38961e-958c-4d56-b095-39783d05a6bf.png">